platform_umbrella/apps/common_core/lib/common_core/resources/sso/sso.ex

```diff
@@ -5,7 +5,7 @@ defmodule CommonCore.Resources.SSO do
   alias CommonCore.StateSummary
   alias CommonCore.StateSummary.Batteries
 
-  @proxy_enabled_batteries ~w[notebooks smtp4dev vm_agent vm_cluster]a
+  @proxy_enabled_batteries ~w[notebooks smtp4dev vm_agent victoria_metrics]a
```

Comment:
The battery type changed but it wasn't changed here so the auth provider in the mesh config wasn't getting created.

```
error	authorization	Processed authorization policy: failed to process CUSTOM action, will generate deny configs for the specified rules: available providers are [smtp4dev-ext-authz-http vm-agent-ext-authz-http] but found "victoria-metrics-ext-authz-http"
```